tools/codegen/pkg/bootstrapgen/generator.go

```diff
@@ -32,16 +28,16 @@ func (g *Generator) Generate(_ []string) error {
 	// TODO Complete the imlementation here.. Next PR.
 	// ..
 
-	f1, err := os.Create(g.OutFilePath)
-	if err != nil {
-		return err
-	}
-	defer func() { _ = f1.Close() }()
-	if _, err = f1.Write([]byte{}); err != nil {
-		_ = f1.Close()
-		_ = os.Remove(f1.Name())
-		return err
-	}
+	//f1, err := os.Create(g.OutFilePath)
```

Comment:
this was in-progress code with a TODO. felt better to not delete, but still needed to do something to make linters happy.